Admin/MenuNodeAdmin.php

```diff
@@ -160,10 +160,10 @@ public function getFormBuilder()
      */
     public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
     {
-        $menuNodeNode = parent::buildBreadcrumbs($action, $menu);
+        $menuNodeItem = parent::buildBreadcrumbs($action, $menu);
```

Comment:
agreed. but why not call it simply menuItem? nodenode and nodeitem both feel strange...